### ISSUE TITLE
Atom crashes after using jekyll-atom

### ISSUE THREAD

**Issue by SalGnt** [Arcath/jekyll-atom#12]
If I try to use the `Window: Reload` command Atom crashes. This happens only after I used this plugin (e.g. after using of `Jekyll: Manage` or `Jekyll: Toolbar`).

I'm using Atom Version `0.177.0`.

**Comment by Arcath** [comment]
If you open dev tools `Alt-Ctrl-I`, go to the console and tick `preserve log` then trigger `Window: Reload` what does it log?

**Comment by SalGnt** [comment]
I don't see anything logged, Atom crashes immediately and when I relaunch it I have a clean console.

**Comment by Arcath** [comment]
is the server running when your restarting atom?

**Comment by SalGnt** [comment]
No. If you want to reproduce the crash you should follow these steps:

1. Launch Atom;
2. Open the Toolbar or the Manage tab;
3. Run the `Window: Reload` command.

**Comment by Arcath** [comment]
I have performed that test on my Mac and PC and had no issues

**Comment by SalGnt** [comment]
I recorded a [video](http://cl.ly/3E2S2X1c1O1C), please take a look at it.

**Comment by SalGnt** [comment]
Ok, I got something!

```
Uncaught TypeError: undefined is not a function

/Applications/Atom.app/Contents/Resources/app/node_modules/deprecation-cop/lib/deprecation-cop-view…:149 
module.exports.DeprecationCopView.serialize

/Applications/Atom.app/Contents/Resources/app/src/pane.js:67
(anonymous function)

/Applications/Atom.app/Contents/Resources/app/src/pane.js:66
module.exports.Pane.serializeParams

/Applications/Atom.app/Contents/Resources/app/node_modules/serializable/lib/serializable.js:79 
module.exports.Serializable.serialize

/Applications/Atom.app/Contents/Resources/app/src/pane-container.js:84 
module.exports.PaneContainer.serializeParams

/Applications/Atom.app/Contents/Resources/app/node_modules/serializable/lib/serializable.js:79 
module.exports.Serializable.serialize

/Applications/Atom.app/Contents/Resources/app/src/workspace.js:146 
module.exports.Workspace.serializeParams

/Applications/Atom.app/Contents/Resources/app/node_modules/serializable/lib/serializable.js:79 
module.exports.Serializable.serialize

/Applications/Atom.app/Contents/Resources/app/src/atom.js:639 
module.exports.Atom.unloadEditorWindow

/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:89
(anonymous function)

/Applications/Atom.app/Contents/Resources/app/src/space-pen-extensions.js:115 
handler

/Applications/Atom.app/Contents/Resources/app/node_modules/space-pen/vendor/jquery.js:4681 
jQuery.event.dispatch

/Applications/Atom.app/Contents/Resources/app/node_modules/space-pen/vendor/jquery.js:4359 
elemData.handle
```

**Comment by Arcath** [comment]
I released jekyll-atom 0.4.3 yesterday which has a bunch of deprecation fixes in it, does that solve this issue?

**Comment by SalGnt** [comment]
Everything seems to work fine now, thank you!
